.github/workflows/gitlab-runners-ubi8-git-pr.yaml

```diff
@@ -0,0 +1,31 @@
+name: ubi8-git-pr-ci
+on:
+  pull_request:
+    paths:
+      - gitlab-runners/ubi8-git/**
+jobs:
+  build:
+    env:
+      context: gitlab-runners/ubi8-git
+      image_name: gitlab-runners-ubi8-git
+    # Only build on redhat-cop repo, do not attempt to build on forks
+    if: github.repository == 'redhat-cop/containers-quickstarts'
```

Comment:
why not make the repo a variable and get rid of this IF that way people can test building images with the action on their forks by setting up the destination repo variable?

Reply:
Good point and for the PR workflow there's no need for the repo at all as we're not pushing anywhere. The IF can still be removed and you would trigger this workflow by opening a PR against your own fork (with changes in `gitlab-runners/ubi8-git/**`).